challenge/src/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class Button extends React.Component {

```

Comment:
perhaps try using a functional component, a class component works here, but it doesn't really need any life cycle methods so a functional component may be cleaner